prody/proteins/interactions.py

```diff
@@ -2219,36 +2422,38 @@ def calcProteinInteractionsDCD(self, atoms, trajectory, filename=None, **kwargs)
         return HBs_nb, SBs_nb, RIB_nb, PiStack_nb, PiCat_nb, HPh_nb
 
 
-    def getInteractions(self, filename=None, **kwargs):
+    def getInteractions(self, **kwargs):
         """Return the list of all interactions.
         
-        :arg filename: name of DAT file where interactions will be saved.
-        :type filename: str
-        
         :arg selection: selection string
         :type selection: str
     
         :arg selection2: selection string
-        :type selection2: str """
+        :type selection2: str 
+            
+        Selection:
+        If we want to select interactions for the particular residue or group of residues: 
+            selection='chain A and resid 1 to 50'
+        If we want to study chain-chain interactions:
+            selection='chain A', selection2='chain B'  """
 
         if len(kwargs) != 0:
```

Comment:
If filename is in the kwargs then this may not work as expected. Perhaps do the following first:

filename = kwargs.pop(“filename”, None)